tf_agents/train/actor.py

```diff
@@ -122,7 +122,13 @@ def __init__(self,
           max_steps=steps_per_run,
           max_episodes=episodes_per_run)
     elif isinstance(env, tf_environment.TFEnvironment):
-      raise ValueError("Actor doesn't support TFEnvironments yet.")
+      self._driver = tf_driver.TFDriver(
```

Comment:
Observers may not work with TFEnvironments since those are typically batched. Can you add a test to make sure this change works?